Hestia/app/src/main/java/hestia/UI/dialogs/IpDialog.java

```diff
@@ -49,4 +51,9 @@ void pressConfirm() {
                     Toast.LENGTH_SHORT).show();
         }
     }
+
```

Comment:
I suggest we create a new method that checks the entered ip address. Such a check would include the following:
- ip is not null, nor is it empty.
- should contain only numbers.
- should have the correct formating of an ip (v4/v6, groups of numbers not greater than 255)